server/src/main/java/org/opensearch/indices/replication/copy/ReplicationCheckpoint.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices.replication.copy;
+
+import org.opensearch.common.Nullable;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.index.shard.ShardId;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Represents a Replication Checkpoint which is sent to a replica shard.
+ *
+ * @opensearch.internal
+ */
+public class ReplicationCheckpoint implements Writeable {
```

Comment:
I think this ReplicationCheckpoint class belongs in the replication.checkpoint package, not the copy package.

Reply:
Thanks, made the change